docs/src/prototypes/chatPane/chatPaneContent.tsx

```diff
@@ -46,6 +74,14 @@ class ChatPaneContainer extends React.PureComponent<ChatPaneContainerProps> {
       scrollRef.scrollToBottom()
     }
   }
+
+  private getMessagePreviewForScreenReader(props) {
```

Comment:
we should think about making this to be a part of Chat's API, so that these customization means will be available to the client (as logic of this method is item-specific)